chain/chain/src/flat_storage_resharder.rs

```diff
@@ -432,6 +452,7 @@ impl FlatStorageResharder {
             }
 
             num_batches_done += 1;
+            metrics.set_split_shard_processed_batches(num_batches_done);
```

Comment:
Can you also write the batch size and size_done? The batches depend on config so it's only meaningful combined with the batch size. The size done can probably be derived from the previous two but it would also be easier to see it as a separate metric in grafana.

Reply:
Good idea, I'll add those two metrics!